pom.xml

```diff
@@ -154,16 +155,25 @@
                 <hdp_version>2.6.5.0</hdp_version>
                 <global_hbase_version>1.1.1</global_hbase_version>
                 <global_hbase_guava_version>12.0</global_hbase_guava_version>
+                <!--<global_kafka_version>0.10.0.1</global_kafka_version>-->
+                <!--<global_slf4j_version>1.7.7</global_slf4j_version>-->
+                <global_kafka_version>2.0.0</global_kafka_version>
+                <global_slf4j_version>1.7.25</global_slf4j_version>
+                <global_storm_version>1.2.1</global_storm_version>
             </properties>
         </profile>
         <profile>
             <id>HDP-2.5.0.0</id>
             <properties>
                 <hdp_version>2.5.0.0</hdp_version>
                 <build_number>1245</build_number>
-                <global_storm_kafka_version>1.2.2</global_storm_kafka_version>
-                <global_storm_version>${base_storm_version}.${hdp_version}-${build_number}</global_storm_version>
-                <global_kafka_version>${base_kafka_version}.${hdp_version}-${build_number}</global_kafka_version>
+                <!--<global_storm_kafka_version>1.2.2</global_storm_kafka_version>-->
+                <!--<global_storm_version>${base_storm_version}.${hdp_version}-${build_number}</global_storm_version>-->
+                <!--global_kafka_version>${base_kafka_version}.${hdp_version}-${build_number}</global_kafka_version-->
+                <!--global_slf4j_version>1.7.7</global_slf4j_version-->
```

Comment:
Should we leave the existing HDP-2.5.0.0 profile as-is until we can remove it as a separate PR?

Reply:
The latest commit leaves the HDP-2.5.0.0 profile as is.  I set the HDP-3.1 profile active by default and added all the versions that have been upgraded so far.  We can move dependencies over from the main property section as we upgrade them.  Does that work?

Reply:
I just tested full dev and it passed the build step.

Reply:
That works thanks.